### PR TITLE
style(tabs): add styling to tabs

### PR DESCRIPTION
**Pull request by SAHU-01**
**Notes for Reviewers**

This PR adds styling to sistent tabs.

image

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
@SAHU-01 can you share some screenshot how does it look?

File: src/base/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -1,7 +1,33 @@
 import { Tabs as MuiTabs, type TabsProps as MuiTabsProps } from '@mui/material';
 
 export function Tabs(props: MuiTabsProps): JSX.Element {
-  return <MuiTabs {...props} />;
+  return (
+    <MuiTabs
+      {...props}
```
Comment (sudhanshutech) [inline src/base/Tabs/Tabs.tsx:6]: don't style it here, style under theme.modifier components

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Thanks for this feedback, @sudhanshutech

File: src/base/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -1,3 +1,37 @@
+// import { Tabs as MuiTabs, type TabsProps as MuiTabsProps } from '@mui/material';
+
+// export function Tabs(props: MuiTabsProps): JSX.Element {
+//   return (
+//     <MuiTabs
```
Comment (sudhanshutech) [inline src/base/Tabs/Tabs.tsx:5]: why its being commented and not removed?

File: src/theme/components/tab.modifier.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Components, Theme } from '@mui/material';
+import { GRAY } from '../colors';
+
+export const MuiTab: Components<Theme>['MuiTab'] = {
+  styleOverrides: {
+    root: ({ theme }) => ({
+      backgroundColor: 'transparent',
+      color: theme.palette.text.primary,
```
Comment (sudhanshutech) [inline src/theme/components/tab.modifier.ts:8]: do we not have any token to use it in here?

File: src/theme/components/tab.modifier.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      '&.Mui-selected': {
+        backgroundColor: 'transparent',
+        color: theme.palette.mode === 'dark' ? GRAY : GRAY
```
Comment (sudhanshutech) [inline src/theme/components/tab.modifier.ts:11]: same here?

File: src/base/Tabs/Tabs.tsx (hunk 2 above)
Comment (SAHU-01) [reply src/base/Tabs/Tabs.tsx]: My apologies, I've removed the same.

File: src/theme/components/tab.modifier.ts (hunk 4 above)
Comment (SAHU-01) [reply src/theme/components/tab.modifier.ts:11]: Sure, I've set it to none. I couldn't find a token that makes it transparent so I updated it to 'none'.

**Review by sudhanshutech — COMMENTED** [review]
@SAHU-01 don't use hard code colors names like gray or white instead use tokens

**Comment by SAHU-01** [comment]
> @SAHU-01 don't use hard code colors names like gray or white instead use tokens

Thank you for highlighting I'll add these today and update the PR.

**Comment by sudhanshutech** [comment]
@SAHU-01 i have added the theme for tabs , test that in meshery and if everything looks good.

**Pull request closed without merging**